packages/docs/docs/advanced/bank-sync/enablebanking.md

```
@@ -0,0 +1,16 @@
+# Enable Banking Setup
+
+Start by enabling the **experimental feature** by going to Settings -> Advanced Settings -> Experimental Features -> Enable Banking.
+
+To acquire the API key create and log in to your account at: https://enablebanking.com/sign-in/
+
+Create a new application: https://enablebanking.com/cp/applications, select **production** and make sure the redirect url is `https` and uses your domain.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)